gustaf/spline/visualize.py

```diff
@@ -128,41 +212,41 @@ def _vedo_showable(spline):
     res = enforce_len(
```

Comment:
Default value for lighting is set in l. 202

Reply:
Resolved the issue